assets/src/theme/balancedVar.js

```diff
@@ -0,0 +1,41 @@
+const balanced = require( 'balanced-match' );
```

Comment:
This was entirely copied from https://github.com/greenpeace/planet4-postcss-css-variables/blob/master/lib/balanced-var.js, still need to make it use `import` syntax.